common/authorization/interceptor_test.go

```diff
@@ -133,3 +133,23 @@ func (s *authorizerInterceptorSuite) TestAuthorizationFailed() {
 	s.Nil(res)
 	s.Error(err)
 }
+
+func (s *authorizerInterceptorSuite) TestNoopClaimMapperWithoutTLS() {
```

Comment:
Would this test fail without this change?

Reply:
yes